src/components/AddGithubStarModal.svelte

```diff
@@ -0,0 +1,91 @@
+<script>
+  /* eslint-disable no-undef */
+  import Modal from './Modal'
+  import Icon from './Icon/Icon'
+
+  const isInExtension = typeof chrome !== 'undefined' && chrome.tabs !== undefined
+
+  let asAlreadyClicked = true
+  $: isModalOpen = !asAlreadyClicked
+
+  const asClickedKey = 'has-clicked'
+  const openCounterKey = 'popup-open-counter'
+  const minOpenCounter = 5
+
+  const setAsClickedStorage = () => {
+    chrome.storage.local.set({ [asClickedKey]: true })
+  }
+
+  const setOpenCounterStorage = (count) => {
```

Comment:
Pourquoi utiliser un compteur et pas simplement un booléen ?
Dès que l'utilisateur a cliqué sur la croix, la pop-up ne devrait plus s'afficher !

Reply:
Because if a new user install the extension he will directly see the modal which isn't user friendly.

Reply:
Oh ! Right 👍
I misunderstood